Header header
uint8 MoveSelected
uint8 DeleteSelected
uint8 DeleteAll
uint8 MeasureLength
uint8 EstimateLine
uint8 EstimatePlane
uint8 EstimateCircle
uint8 EstimateRigidTransform
uint8 Crop
uint8 Undo
uint8 DisplayIndices
uint8 DisplayConnections
float64 CursorMaxDistance
float64 LineRoughMaxDistance
float64 LineMaxDistance
float64 PlanePrismDistance
float64 PlaneMaxDistance
float64 PlaneHullScaleFactor
uint8 PlaneHullCrop
uint8 PlaneFitTwice
float64 CircleMinRadius
float64 CircleMaxRadius
float64 CircleMaxDistance
float64 CirclePlanePrismDistance
